drivers/volume/volume.go

```diff
@@ -150,6 +165,14 @@ type NodeInfo struct {
 	Status NodeStatus
 }
 
+// ClusterDomainsInfo is a list of active and inactive cluster domains
+type ClusterDomainsInfo struct {
```

Comment:
Can we not use the existing `ClusterDomains` type here? It is identical to this new struct.

Reply:
done